charts/plex-media-server/templates/statefulset.yaml

```diff
@@ -114,14 +114,24 @@ spec:
         - containerPort: 32400
           name: pms
         env:
-        {{- if .Values.ingress.enabled }}
+        {{- $url := .Values.ingress.url }}
```

Comment:
These conditionals are a bit hard to follow, maybe this can be refactored such that:

1. we keep the `ADVERTISE_IP` default value as `.Values.ingress.url :443`
2. `if hasKey .Values.extraEnv "ADVERTISE_IP"` then use that as the value for `env.ADVERTISE_IP`